nmkapp/pipeline.py

```diff
@@ -0,0 +1,9 @@
+from social_core.pipeline.partial import partial
+from django.shortcuts import redirect
+
+@partial
+def request_display_name(strategy, details, user=None, is_new=False, *args, **kwargs):
+    if user and user.first_name:
+        return
+    else:
+        return redirect('request_display_name')
```

Comment:
Koristi reverse() f-ju ovde, umesto da hardcodeuješ putanju (i ja sam to radio, ali polako prelazim:) na proper way:)